homeassistant/components/velbus/sensor.py

```diff
@@ -53,9 +53,9 @@ def __init__(
             self._attr_name = f"{self._attr_name}-counter"
         # define the device class
         if self._is_counter:
-            self._attr_device_class = SensorDeviceClass.ENERGY
-        elif channel.is_counter_channel():
             self._attr_device_class = SensorDeviceClass.POWER
+        elif channel.is_counter_channel():
+            self._attr_device_class = SensorDeviceClass.ENERGY
```

Comment:
Is this a breaking change?

Reply:
this was a reported bug, so this is more a bugfix

Reply:
Oh sorry, it looked like it was just moved, but indeed the logic changed. 

@Cereal2nd Normally these should have been split into separate PRs. Please take that into account for next time.